src/test/resources/database/remove-saved-category-from-db.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM categories WHERE name = 'Verse';
```

Comment:
i'd suggest forming a habit and always reviewing your PR to fix no empty line at EOF issues
![image](https://github.com/mhorianin/book-store-web/assets/49767049/65a29244-4a57-4fe7-b966-b850c84d1d2f)